examples/sam2_amg_server/README.md

```diff
@@ -41,7 +41,26 @@ The 'ao' mode is a copy of the baseline with modifications to make the code more
 ### 0. Download checkpoints and install requirements
 
 ```
-pip install -r requirements.txt
+# From the top-level "ao" directory
+
+# Create and activate a virtual environment
+python -m venv venv && source venv/bin/activate
+
+# Install requirements for this example
+pip install -r examples/sam2_amg_server/requirements.txt
+
+# Install torch and ao nightlies
+#   (try to uninstall first to be safe, use CUDA 12.1 to be safe)
+pip uninstall torch torchao
+
+pip install --pre torch torchvision torchaudio --index-url https://download.pytorch.org/whl/nightly/cu121
+pip install --pre torchao --index-url https://download.pytorch.org/whl/nightly/cu121
+
+# Build from source
+python setup.py develop
```

Comment:
This won't be necessary if the user installed torchao nightly above. This was really just needed to pull in the very latest change and/or to make changes locally and push them more easily. Also I think we had  uninstall torchao from above again so that this works.

Reply:
Yeah for now that seems necessary. I also found that the nightly package doesn't include the config files in https://github.com/pytorch/ao/tree/main/torchao/_models/sam2/configs so it's kind of the only choice.